wgs/wg-pipelines/wg-charter-pipelines.md

```diff
@@ -0,0 +1,96 @@
+# ML Pipelines Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+The goal of the Kubeflow Pipelines project is to build a Kubernetes optimized stack for creating and deploying ML Pipelines and enabling a rich ecosystem of pipelines components and tools. ML Pipelines project has the goal of enabling simple and reliable productionalization of autonomous ML workflows, advancing the standards of ML Engineering and making them available to a large community of users across various Kubernetes deployment options.
+
+
+### In scope
+
+#### Code, Binaries and Services
+
+##### Backend Servers
+- ML Pipelines user facing APIs 
+- Storage abstraction and integration components for pipelines
+- Pipelines Orchestration Engine
+Includes Argo and Tekton based forks, as well as new versions of orchestrators
+- Metadata Store that tracks pipeline artifacts, their lineage and dependencies as well as the ML Metadata Service exposing the Metadata APIs, and the Metadata collection agent
+- Pipeline job scheduling controller, and continuous pipeline execution services
+
+##### FE and UI
+- Pipelines Templates UI
+- Experiment tracking and run exploration UI
+- Metadata Tracking and Lineage Exploration UI
+- Metadata metric visualization 
+
+##### Tools
+- Authoring SDK for components and pipelines
+- Tools for pipeline creation and deployment
+- CLI for interacting with Pipelines API
+
+##### Deployment
+- Deployment scripts and tools for deploying ML Pipeline service components on Kubernetes engines of various cloud providers and on-prem deployments
+
+##### Pipeline Content
+- Cloud specific integration components
+- HW and SW vendor specific components
+- Pipeline examples for different ML use cases
+
+##### Documentation
+- Concepts documentation 
+- Cloud provider specific documentation
+- Documentation for the SDK and REST API
+- Documentation for the sample components and pipelines 
+
+
+#### Dependencies
+
+ML Pipeline projects may have dependencies on the following projects managed by other KF work groups:
+- KF User Profiles Controller
+- KF Central UI Console
+- Notebook and KF integration of Notebooks
+- TF Job and other training controllers
```

Comment:
Can we also add AutoML dependency?

```suggestion
- TF Job and other training controllers
- Kubeflow AutoML experiments
```

Reply:
In what scenarios this dependency would materialize?

Reply:
I think Andrey means Katib, right?
https://github.com/kubeflow/pipelines/pull/754/files

By the way, we also have other options for HP tune etc., e.x. in-trainer Keras Tunner, out-trainer TFX Tunner component, [Google Cloud HP Tuning](https://cloud.google.com/ai-platform/training/docs/using-hyperparameter-tuning), [Google Cloud AutoML](https://cloud.google.com/automl) etc. 

Here just list the other Kubeflow components per the context.

Reply:
Yes, thanks @rmgogogo.
This launcher is a bit outdated, we will update launcher with new v1beta1 version after we release it.